Jamie_Keng/skillset_challenge.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center> Statistical Arbitrage with Vine Copulas - Pair Selection </center>\n",
+    "\n",
+    "Author: Jui Lin(Jamie) Keng <br>\n",
+    "Email: jamiekeng1016@gmail.com <br>\n",
+    "LinkedIn : https://www.linkedin.com/in/jui-lin-jamie-keng-3883b6131/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
It would have been good to explain each approach. Your notebook is missing a body section.